test/bdd/steps/hooks.js

```diff
@@ -61,10 +68,25 @@ After(function (testCase, done) {
     done();
 });
 
-AfterAll(() => {
-    // TODO: After-all.
-    // Delete all Arango dbs.
-    // Drop all data to artifacts.
+AfterAll(async function () {
+    // Delete almost all Arango dbs.
+    const systemDb = new Database();
+    systemDb.useBasicAuth(config.database.username, config.database.password);
+
+    const listOfDatabases = await systemDb.listDatabases();
+
+    listOfDatabases.forEach(async function (databaseItem) {
+        if (databaseItem !== '_system' && databaseItem !== 'origintrail' && databaseItem !== 'origintrail-develop' && databaseItem !== 'origintrail-staging' && databaseItem !== 'origintrail-stable') {
```

Comment:
I think it's enough to check for:
```
databaseItem.includes('origintrail-test-')
```
or to be more accurate you can test for whole name:
```
databaseitem.match(/^origintrail-test-[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/gi).length > 0
```